Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -1962,6 +2043,155 @@ def cancel_containment_command(client: Client, args: Dict[str, Any]) -> CommandR
     return CommandResults(readable_output=message)
 
 
+"""
+HOST SETS
+"""
+
+
+def delete_host_set_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    host_set_id = args.get('host_set_id')
+
+    if not host_set_id:
+        raise ValueError('Host Set ID is required')
+
+    try:
+        client.delete_host_set_request(host_set_id)
+        message = f'Host set {host_set_id} was deleted successfully'
+    except Exception as e:
+        if '404' in str(e):
```

Comment:
just to make sure: status code will be 200 even when it's actually a failure?

Reply:
![image](https://user-images.githubusercontent.com/93524335/178439980-d47a7e67-66e2-4971-a5d3-d2e5fbc0a54e.png)
According to the documentation, those are the only 2 error codes that can return

Reply:
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/93524335/178476879-f03e143c-d48e-4605-ab28-f62c33855019.png">
According to the documentation, there are only 2 possible status codes